cluster/gce/gci/configure-helper.sh

```diff
@@ -2218,6 +2218,13 @@ EOF
     local -r metadata_proxy_yaml="${dst_dir}/metadata-proxy/gce/metadata-proxy.yaml"
     update-prometheus-to-sd-parameters ${metadata_proxy_yaml}
   fi
+  if [[ "${ENABLE_ISTIO:-}" == "true" ]]; then
+    if [[ "${ISTIO_AUTH_TYPE:-}" == "MUTUAL_TLS" ]]; then
```

Comment:
Humm, I was under the impression that we also need to pipe these envs into build-kube-env:
https://github.com/kubernetes/kubernetes/blob/9b646503113ae9eb33e57ca231edb868247ba1f5/cluster/common.sh#L472

Does it just work?

Reply:
I manually tested by just temporarily setting the variables directly in the script itself. This will work for GKE which has an internal path for setting env vars but it will be good to implement the non-GKE path as well. I'll take a look at that.

Reply:
I see, yeah it would be great to have a common path so that we could enable it in CI jobs as well (for GCE cluster).

Reply:
https://github.com/kubernetes/test-infra/pull/6612 is still in progress from @krzyzacy 
Let me see if I can help with that. Seems like it would be generally useful functionality. 
The public API and CLI are being updated right now (ETA ~2 weeks) so that would be the way to enable it in tests when it becomes available.